src/index.css

```diff
@@ -15,10 +19,150 @@ code {
 html{
   font-size: 125%;
   text-align: center;
+  background-image: url(../assets/redlightbackground\ \(2\).jpg);
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
+h1,
+label,
+p,
+h2,
+h3{
+  color: white;
+}
+
+button{
+  background-color: white;
+  border: white 3px solid;
+  color: black;
+  transition: 0.5s;
+}
+
+button:hover{
+  background-color: lightcoral;
+  border: solid lightcoral 3px;
```

Comment:
You can try to add a `cursor: pointer;` so that users know that it's a clickable element :)